docs/sec_intro.rst

```diff
@@ -13,7 +13,16 @@ pip install mpl_data_cast
 The :ref:`Command-line interface <sec_cli>` is available via the ``mpldc``
 command.
 
-A graphical user interface, including installers for Windows and macOS, is planned.
+A graphical user interface (GUI), including installers for Windows and macOS,
+is currently under development. An initial GUI to use MPL-Data-Cast for
+transfering RT-DC data is already available. If you encounter bugs or have
+suggestions for improvement, open an issue on
+`GitHub <https://github.com/GuckLab/MPL-Data-Cast/issues>`_.
+To start the GUI, run
+
+```
+python -m mpl_data_cast
```

Comment:
Since we have a CLI, we should probably distinguish between the two and only allow

```
python -m mpl_data_cast.cli
```
and 

```
python -m mpl_data_cast.gui
```

That means you would have to move the `__main__.py` file to the `gui` directory and create such a file in the `cli` directory as well. It should work.

Also note that the `mpldc` and `mpl_data_cast` programs are registered in `setup.py` which also requires cleaning-up. I would propose `mpldc` and `mpldc-gui`.